.github/workflows/run-test-cases.yml

```diff
@@ -296,60 +307,23 @@ jobs:
       # For push and release, we need to wait for the Helm chart and
       # associated containers to build.
       - if: github.event_name == 'push' || github.event_name == 'release'
-        name: Set sleep duration before running script to 2700
-        run: echo 2700 > /tmp/sleep_duration.txt
+        name: sleep before running script for 2700s
+        run: sleep 2700
 
-      # For pull_request, use the locally built containers.
-      - if: startsWith(github.event_name, 'pull_request')
-        name: Tell Helm to use the 'local' labels for container images
+      - name: Execute test script
+        working-directory: ./test/e2e
         run: |
-          git fetch origin main
-          git show origin/main:version.txt > /tmp/version_to_test.txt
-          echo '--set agent.image.pullPolicy=Never,agent.image.tag=pr-amd64,controller.image.pullPolicy=Never,controller.image.tag=pr-amd64,webhookConfiguration.image.pullPolicy=Never,webhookConfiguration.image.tag=pr-amd64' > /tmp/extra_helm_args.txt
-      # For non-PR (i.e. push, release, manual), version.txt is corresponds
-      # to an existing Helm chart.
-      - if: (!(startsWith(github.event_name, 'pull_request')))
-        name: Use current version for push
-        run: cat version.txt > /tmp/version_to_test.txt
+          case "${{ github.event_name }}" in "push");; "release") extra_param="--release";; *) extra_param="--use-local";; esac
```

Comment:
We may need to ensure that on releases, this only runs once `run-helm.yaml` completes, otherwise this may pull the previous releases charts

Reply:
The `run-helm` is not the only one we need to wait for, there are also all the `build-.*-container`

I used the same way as before for that (just not at the same place), a.k.a `sleep 2700`, it now is in an action step, while it was in the python script before.

(quite unrelated but made me notice I didn't change the paths to look for to trigger e2e test run, so will do that)